src/+otp/+arenstorf/+presets/Canonical.m

```diff
@@ -1,17 +1,36 @@
 classdef Canonical < otp.arenstorf.ArenstorfProblem
+    %CANONICAL One period of a satellite moving in an Earth-Moon system
+    %   This configuration can be found in
+    %
+    %   Hairer, Ernst, et al. Solving Ordinary Differential Equations I:
+    %   Nonstiff Problems. 2nd ed., vol. 8, Springer-Verlag, 1993, pp. 129-130.
+    %
+    %   See also otp.arenstorf.ArenstorfProblem
+    
     methods
-        function obj = Canonical(varargin)
-                 
-            p = inputParser;
-            p.addParameter('m1', 0.012277471);
-            p.parse(varargin{:});
-            s = p.Results;
+        function obj = Canonical(mu)
+            %CANONICAL Construct a canonical Arenstorf problem
+            %   OBJ = CANONICAL(MU) Uses a Moon of mass MU. The time span and
+            %   initial conditions use the same data type as MU. Single
+            %   precision, vpa, and other types are supported.
+            %
+            %   OBJ = CANONICAL() Uses a MU corresponding to the Moon
+    
+            if nargin < 1
+                mu = 0.012277471;
+            end
             
-            params.m1 = s.m1;
-            params.m2 = 1-params.m1;
+            params = otp.arenstorf.ArenstorfParameters;

```

Comment:
Does octave not support input parser?

Reply:
It does, but it seemed unnecessary for a one-input constructor.

Reply:
Sure